bevy_lint/docs/README.md

```diff
@@ -0,0 +1,64 @@
+# Contributor's Guide
+
+Thanks for your interest in contributing to `bevy_lint`! Please feel free to skim through the following table of contents:
+
+- [Tutorials](tutorials/)
+- [How-to](how-to/)
+    - [How to Work with Types](how-to/types.md)
+    - [How to Release `bevy_lint`](how-to/release.md)
```

Comment:
```suggestion
    - [How to Release `bevy_lint`](how-to/release.md)
    - [Configuring rust-analyzer for rustc](https://rustc-dev-guide.rust-lang.org/building/suggested.html#configuring-rust-analyzer-for-rustc)
```

Reply:
In this case, I'm leaning more towards writing a `bevy_lint`-specific tutorial on setting up the editor. (But probably in a separate PR!)

Reply:
I use NeoVim so it does not pick up the `.vscode` folder out of the Box and therefore `"rust-analyzer.rustc.source": "discover"` is not set.  But I can add a section on how to solve this if needed